src/lib/ical/utils.ts

```diff
@@ -307,10 +307,18 @@ export function makeAcmEvent(
 
   const { description, variables } = parseDescription(icalEvent['DESCRIPTION']);
 
-  const { location, meetingLink } = parseLocation(
-    icalEvent['LOCATION'],
-    variables.get('ACM_LOCATION')
-  );
+  const locations = parseLocation(icalEvent['LOCATION'], variables.get('ACM_LOCATION'));
+
+  let location = locations.location;
+  const meetingLink = locations.meetingLink;
+
+  const hosted = ['Discord', 'Zoom'];
+
+  location = location?.trim()
+    ? hosted.includes(location.trim())
+      ? `Hosted on ${location}`
+      : location
+    : 'TBD';
```

Comment:
Honestly if we're gonna keep the Hosted on, I think using an if statement would better help readability in this case.
```
if (hosted.includes(location.trim()))
  location = `Hosted on ${location}`
```

Also no need for the TBD or trim here because shouldn't event-item already trim it and/or make it a TBD? In which case why not move this logic back to event-item?

Reply:
We should keep the logic in here so we can have testing done to this and have expected results, I'll make a quick change to make it more readable though

Reply:
I feel like our formatting operation, prepending “Hosted on”, could be handled in the Svelte file. Off-topic, but Angular has [pipes](https://angular.io/guide/pipes) for the technique I’m describing. 

The alternative is to add a new property to the `AcmEvent` interface, other than `AcmEvent.location`, which stores the formatted hosted-by string, something like `AcmEvent.hostedBy`.